services/gateway/apis/http-version1/methods/newsfeed.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const newsfeedSource = require('../../../sources/version1/newsfeed');
+const envelope = require('../../../sources/version1/mappings/stdEnvelope');
+const { transformParams, response, getSwaggerDescription } = require('../../../shared/utils');
+
+module.exports = {
+	version: '2.0',
+	swaggerApiPath: '/market/newsfeed',
+	rpcMethod: 'get.newsfeed.articles',
+	tags: ['Newsfeed'],
+	params: {
+		source: { optional: true, type: 'string', min: 1, pattern: /^\b[a-z]+_[a-z]+\b$/ },
```

Comment:
Should it support multiple sources (as CSV)? Currently, it only supports one source at a time.

Reply:
I am updating to support multiple sources.